### DISCUSSION TITLE
Use actual entity name instead of `Model`

### DISCUSSION THREAD

**Discussion by cloneable** [SeaQL/sea-orm#2075] [Q&A]
Hi, is there a plan to allow entity structs to be generated with their actual names before v1 or is this pretty much set in stone now? I just started using SeaORM and it's a bit frustrating when it's just `Model`, `Option<Model>` etc. everywhere. IDE and rust-analyzer are not of much help.

**Comment by shanesveller** [comment]
For your own modules where that kind of name is unhelpful, have you considered writing `use crate::whatever::Model as SomeBetterNoun` to make the rest of the file more readable?

**Reply by cloneable** [reply]
Thanks! I meant types shown by rust-analyzer, which are usually unqualified. Yes, I can use such aliases for explicit typing. SeaORM even creates them. But this doesn't help with information from type inference. I believe DX will benefit a lot from distinct model names.
